frontend/packages/core/src/button.tsx

```diff
@@ -102,7 +114,7 @@ const StyledBorderButton = styled(StyledButton)({
 });
 
 /** Provides feedback to the user in regards to the action of the button. */
-type ButtonVariant = "neutral" | "primary" | "danger" | "destructive";
+type ButtonVariant = "neutral" | "primary" | "danger" | "destructive" | "tertiary";
```

Comment:
wouldn't it be secondary?

Reply:
secondary is our "neutral" color palette

Reply:
i would update the design system or the code to match (would prefer the former). no reason to have tertiary without secondary in the impl.

Reply:
okay I'll update this new palette name to be "secondary" and leave a comment for Design to rename "secondary" --> "neutral" in the figma file

Reply:
sgtm!

Reply:
updated & left the comments for Design